scripts/build-release.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+import subprocess
+import zipfile
+
+def get_system_info():
+    # Get OS and architecture information
+    os_info = subprocess.run(["uname"], capture_output=True, text=True, check=True).stdout.strip().lower()
+    arch_info = subprocess.run(["uname", "-p"], capture_output=True, text=True, check=True).stdout.strip().lower()
+
+    if os_info == "linux":
+        os_info = "unknown-linux-gnu"
+    elif os_info == "darwin":
+        os_info = "apple-darwin"
+
+    if arch_info == "arm":
+        arch_info = "aarch64"
+
+    return arch_info, os_info
+
+def build_and_move(feature, tmp_dir, architecture, os_name):
+    print("\n" + "=" * 50)
+    print(f"BUILDING {feature if feature else 'default'}")
+    print("=" * 50 + "\n")
+
+    zip_prefix = f"kinode-{architecture}-{os_name}"
+    if feature:
+        subprocess.run(["cargo", "+nightly", "build", "--release", "--features", feature], check=True)
+        zip_name = f"{zip_prefix}-{feature}.zip"
+    else:
+        subprocess.run(["cargo", "+nightly", "build", "--release"], check=True)
```

Comment:
hey, as long as you're updating this script, can you add the release settings i prefer to optimize for performance?

they are the following:
```
strip = "symbols"
lto = "fat"
codegen-units = 1
```

i think in command form it would look like this:
```
CARGO_PROFILE_RELEASE_LTO="fat" CARGO_PROFILE_RELEASE_CODEGEN_UNITS=1 CARGO_PROFILE_RELEASE_STRIP="symbols" cargo +nightly build --release
```